### ISSUE TITLE
Unknown named module: 'MessageQueue' regression in 0.56.0

### ISSUE THREAD

**Issue by no23reason** [facebook/react-native#20114]
- [x] Review the documentation: https://facebook.github.io/react-native
  - [x] Search for existing issues: https://github.com/facebook/react-native/issues
  - [x] Use the latest React Native release: https://github.com/facebook/react-native/releases

## Environment
React Native Environment Info:
    System:
      OS: macOS High Sierra 10.13.5
      CPU: x64 Intel(R) Core(TM) i5-7267U CPU @ 3.10GHz
      Memory: 29.36 MB / 16.00 GB
      Shell: 5.3 - /bin/zsh
    Binaries:
      Node: 8.11.3 - ~/.nvm/versions/node/v8.11.3/bin/node
      Yarn: 1.7.0 - /usr/local/bin/yarn
      npm: 5.6.0 - ~/.nvm/versions/node/v8.11.3/bin/npm
      Watchman: 4.9.0 - /usr/local/bin/watchman
    SDKs:
      iOS SDK:
        Platforms: iOS 11.4, macOS 10.13, tvOS 11.4, watchOS 4.3
      Android SDK:
        Build Tools: 23.0.1, 25.0.0, 25.0.1, 25.0.2, 25.0.3, 26.0.1, 26.0.2, 27.0.1, 27.0.3
        API Levels: 19, 23, 25, 26, 27
    IDEs:
      Android Studio: 3.1 AI-173.4819257
      Xcode: 9.4.1/9F2000 - /usr/bin/xcodebuild
    npmPackages:
      @storybook/react-native: ^3.3.15 => 3.4.2
      @types/react: ^16.3.16 => 16.3.16
      @types/react-native: ^0.55.25 => 0.55.25
      react: 16.4.1 => 16.4.1
      react-native: 0.56.0 => 0.56.0

## Description
After upgrading to `react-native` 0.56 (from the 0.55.4 version) both Android and iOS versions of the app I'm working on started to log this:

> Uncaught (in promise) Error: Unknown named module: 'MessageQueue'
>     at metroRequire (blob:file:///a9fe7d6f-b4b7-4aa0-a31c-d6b4646490ea:64:15)
>     at D.k.forEach.t (file:///Applications/React%20Native%20Debugger.app/Contents/Resources/app.asar/js/RNDebuggerWorker.js:1:13455)
>     at Array.forEach (native)
>     at D (file:///Applications/React%20Native%20Debugger.app/Contents/Resources/app.asar/js/RNDebuggerWorker.js:1:13417)
>     at <anonymous>

This did not happen in the previous versions, hence this is a regression.

## Reproducible Demo
Unfortunately, I'm not at liberty to publish pieces of the app, so I cannot really provide a working repo. I'm happy to answer any questions you might have to help resolve this issue.

**Comment by Denisrudov** [comment]
same for me
and [here](https://github.com/Denisrudov/rn-issue) is the project initial commits

**Comment by mentormate-doug** [comment]
I am hitting this problem as well in my project.  I checked Denisrudov's project and saw that it shows the same error.  I am at a loss for where to look next.  How can I help move this issue forward?

**Comment by omgalbert** [comment]
Same error

**Comment by nicholasrq** [comment]
Same here

react-native-cli: 2.0.1
react-native: 0.56.0
node: v10.5.0
OS: macOS 10.13.5 (17F77)

devices:
physical, Huawei P10 lite, Android 7.0
physical, iPhone X, iOS 11.4.1
emulator, iPhone 6, iOS 11.4
emulator, iPhone 6s, iOS 11.4
emulator, Google Pixel 2, API 26

**Comment by RodolfoSilva** [comment]
Related in this issue https://github.com/jhen0409/react-native-debugger/issues/244

**Comment by luskin** [comment]
We are receiving this error as well, as a workaround it seems this is only an error with the standalone debugger app. If you debug simply at `http://localhost:8081/debugger-ui/` in chrome or whatever you use the error does not occur. Hopefully this will be fixed soon as the standalone app is easier to work with then a browser tab but for now this is fine for us.

**Comment by bwoodlt** [comment]
Having the same error..

**Comment by tonypee** [comment]
yup got it

**Comment by takahiro-saeki** [comment]
Same here. Is there something resolution?

**Comment by simonhintersonnleitner** [comment]
+1

**Comment by mehulmpt** [comment]
Same error

```
react-native: ^0.56
babel and stuff: 7.0.0-beta47
```

**Comment by RodolfoSilva** [comment]
Resolved, update your [react-native-debugger](https://github.com/jhen0409/react-native-debugger/) to new version `v0.7.19`

**Comment by radko93** [comment]
Please let me know if you will still have this issue with the new `react-native-debugger` or without it.

**Comment by mehulmpt** [comment]
Yes this is fixed after updating to v0.7.19. Here's how to quickly do it on mac:
```shell
brew cask reinstall react-native-debugger
```
